native/cocos/platform/android/java/src/com/cocos/lib/CocosEditBoxActivity.java

```diff
@@ -220,6 +220,13 @@ public void onGlobalLayout() {
                         if (!keyboardVisible) {
                             keyboardVisible = true;
                         }
+                        int bottomOffset = 0;
+                        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+                            bottomOffset = getWindow().getDecorView().getRootWindowInsets().getSystemWindowInsetBottom();
```

Comment:
On devices with higher API levels, the system will handle the displacement of the rootView itself. Devices that have been processed will modify the bottom value in mFrameOffsets. In the source code, the bottom value corresponds to getSystemWindowInsetBottom.